src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,41 @@
+import { Button, ListItem, UnorderedList } from "@chakra-ui/react";
+import Link from "next/link";
+import { useDispatch } from "react-redux";
+import { setIsLoggedIn } from "@/src/store/global";
+
+import { DROPDOWN_LINKS, LOGOUT_API } from "./DropdownConstants";
+
+import styles from "./dropdown.module.css";
+
+export function Dropdown() {
+    const reduxDispatch = useDispatch();
+
+    const dropdownItems = DROPDOWN_LINKS.map((link) => {
+        return (
+        <ListItem key={link.id} className={styles.dropdown_items}>
+            <Link href={link.link} className={styles.dropdown_links}>
+                {link.name}
+            </Link>
+        </ListItem>
+        )
+    })
+
+    const logout = () => {
+        fetch(LOGOUT_API, {
+            method: 'GET',
+            credentials: 'include',
+        }).then(() => {
+            reduxDispatch(setIsLoggedIn({isLoggedIn: false}))
+        });
+    }
```

Comment:
Can you create the RTK query or in a separate file for calling API?

Reply:
We can't put it in RTK query because it's a GET request so we can trigger the **queries** on button clicks because it is not a mutation

Reply:
should we move whole function to a separate file or only API call

Reply:
I have move that function in other component and passing it as a props as per vinayak request change please look into latest changes

Reply:
Where is the separate file for this function?

Reply:
Why we need a file for this function it's is not that big and i have moved it to `/dropdown/index.tsx` please have a look